decidim/decidim
# Migrate from babel to esbuild

#### :tophat: What? Why?
This PR is changing babel to esBuild. Doing this change, i could see that we have a ~50% speed improvement on assets compilation time. 

Results:
| Module | Branch | time |
|--|--|--|
| Accountability | develop | webpack 5.89.0 compiled with 5 warnings in **79137** ms |
| Accountability | chore/switch-to-es-build | webpack 5.89.0 compiled with 5 warnings in **45039** ms |

#### Testing
1. Make sure the pipelines are green 
2. Generate a new Development application 
3. Navigate through the pages of the application, admin as well ... and see the application is displaying properly

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
